packages/component-library/package.json

```diff
@@ -13,6 +14,23 @@
     "start:storybook": "serve ./storybook-static -p 6006 --no-request-logging",
     "test:storybook": "test-storybook --url http://127.0.0.1:6006",
     "docker": "docker run -v $PWD/../../:/tests:delegated -w /tests -it --rm --ipc=host mcr.microsoft.com/playwright:v1.20.0-focal /bin/bash"
+=======
+  "version": "3.0.0-alpha1",
```

Comment:
We have to pay attention on how we proceed here. The [changesets documentation](https://github.com/changesets/changesets/blob/main/docs/prereleases.md) states that one must pay attention when doing pre-releases.

Reply:
I think for the initial merge it should work because it is already published. Additional prerelases should be released in a separate branch